popcorndao/workspace
# Add pre commit hook

## Description

This PR adds a husky pre-commit hook. It adds formating of .sol files on save in vs-code.

## To Test
- Check out this branch
- Run yarn install
- make changes on a .tsx or .ts file with formatting issues, stage the file and commit.

**Expected result:**  The file is reformated before committing.
